sources/functions/overseerr

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+overseerr_install() {
+    echo_progress_start "Downloading and extracting source code"
+    dlurl="$(curl -sS https://api.github.com/repos/sct/overseerr/releases/latest | jq .tarball_url -r)"
+    wget "$dlurl" -q -O /tmp/overseerr.tar.gz >> "$log" 2>&1 || {
+        echo_error "Download failed"
+        exit 1
+    }
+    mkdir -p /opt/overseerr
+    tar --strip-components=1 -C /opt/overseerr -xzvf /tmp/overseerr.tar.gz >> "$log" 2>&1
+    echo_progress_done "Code extracted"
+
+    # Changing baseurl before build
+    # export OVERSEERR_BASEURL='/baseurl'
+
+    echo_progress_start "Installing dependencies via yarn"
+    # Ensure sqlite can build right in case it needs to use python
+    if ! which python >> "$log" 2>&1; then #TODO make this a more specific check as this could interfere with other things possibly
+        npm config set python "$(which python3)"
+    fi
+    yarn install --cwd /opt/overseerr >> "$log" 2>&1 || {
+        echo_error "Failed to install dependencies"
+        exit 1
+    }
+    echo_progress_done "Dependencies installed"
+
+    echo_progress_start "Building overseerr"
+    yarn --cwd /opt/overseerr build >> "$log" 2>&1 || {
+        echo_error "Failed to build overseerr sqlite"
+        exit 1
+    }
+    chown -R overseerr: /opt/overseerr
```

Comment:
Not sure why but this is throwing an error for me in testing

```
chown: invalid spec: ‘overseerr:’
```

Odd because this command was already run above :/

Reply:
Ahh, I see now. This particular `chown` command is run before the user is added. This needs to be adjusted so that the user is present